### PR TITLE
French links

### PR DESCRIPTION
**Pull request by keithiancolbourne**
Links on final pages were going to English pages

### CODE REVIEW

File: frontend/src/p2/NextSteps.js [hunk 1]
```diff
@@ -12,107 +13,142 @@ import { Layout } from '../components/layout'
...
+                    Report to your financial institution, if you lost money.
+                  </Trans>
+                </Li>
+                <Li>
+                  <Trans>
+                    Report to the credit bureaus,{' '}
+                    <A href="https://www.consumer.equifax.ca/personal/">
+                      Equifax
+                    </A>{' '}
+                    and <A href="https://www.transunion.ca/">TransUnion</A>, if
```
Comment (sastels) [inline frontend/src/p2/NextSteps.js]: Equifax and TransUnion also have French pages.

Comment (keithiancolbourne) [reply frontend/src/p2/NextSteps.js]: ah, I made a mental note about those but then forgot! Thanks Steve-o

**Comment by sastels** [comment]
oh! it's failing because the test can't find the language in the graphql client. You need to do something like this
https://github.com/cds-snc/report-a-cybercrime/blob/master/frontend/src/components/topbanner/__tests__/TopBanner.test.js#L16

**Comment by sastels** [comment]
:shipit: once things are passing tests 😄

**Pull request merged**